client/control.go

```diff
@@ -292,7 +298,9 @@ func (ctl *Control) msgHandler() {
 		case <-hbSend.C:
 			// send heartbeat to server
 			xl.Debug("send heartbeat to server")
-			pingMsg := &msg.Ping{}
+			pingMsg := &msg.Ping{
+				Timestamp: time.Now().Unix(),
```

Comment:
No need to set this if `AuthenticateHeartBeats` is false. Move it to `SetPing`?

Reply:
@fatedier - do you want the same for NewWorkConn?

Reply:
Yes.

Reply:
I wrote on NewWorkConn - I kind of think Timestamp is useful info in these communication packets. If you disagree I can move it like you suggested

Reply:
Some users feedback that they are sensitive to network traffic in embedded devices. I think we'd better remove it if not used.